src/test/java/seedu/address/logic/commands/AddNotesCommandTest.java

```diff
@@ -0,0 +1,118 @@
+
+        // Create a valid person and set up the model's person list
+        Person validPerson = new PersonBuilder().withNotes().build();
+        ObservableList<Person> personList = FXCollections.observableArrayList(validPerson);
+
+        // Mock the behaviour of ModelMock to return personList when needed
+        when(modelMock.getFilteredPersonList()).thenReturn(personList);
+
+        // Create an ObjectProperty<Person> and mock getFocusedPerson to return it when needed
+        ObjectProperty<Person> focusedPersonProperty = new SimpleObjectProperty<>(validPerson);
+        when(modelMock.getFocusedPerson()).thenReturn(focusedPersonProperty);
+
+        // Setup the valid note and index for use in execution
+        Note validNote = new Note("High profile client.");
+        Index validIndex = Index.fromZeroBased(0);
+
+        AddNotesCommand addNotesCommand = new AddNotesCommand(validIndex, validNote);
+
+        // Execute the addNotesCommand
+        CommandResult commandResult = addNotesCommand.execute(modelMock);
+
+        String expectedOutput = "Added notes to Person: Amy Bee; Phone: 85355255; Email: amy@gmail.com; Address: 123, "
+                + "Jurong West Ave 6, #08-111; Tags: ; Notes: [High profile client.]";
+
+        // Assert that the expected output is matched
+        assertEquals(expectedOutput, commandResult.getFeedbackToUser());
+
+    }
+
+    @Test
+    public void execute_duplicateNote_throwsCommandException() {
+        // Mock the model
+        Model modelMock = mock(Model.class);
+
+        // Create a valid person and set up the model's person list
+        Person validPerson = new PersonBuilder().withNotes("High profile client.").build();
+        ObservableList<Person> personList = FXCollections.observableArrayList(validPerson);
+
+        // Mock the behaviour of ModelMock to return personList when needed
+        when(modelMock.getFilteredPersonList()).thenReturn(personList);
+
+        // Create an ObjectProperty<Person> and mock getFocusedPerson to return it when needed
+        ObjectProperty<Person> focusedPersonProperty = new SimpleObjectProperty<>(validPerson);
+        when(modelMock.getFocusedPerson()).thenReturn(focusedPersonProperty);
+
+        // Setup the valid note and index for use in execution
+        Note validNote = new Note("High profile client.");
+        Index validIndex = Index.fromZeroBased(0);
+
+        AddNotesCommand addNotesCommand = new AddNotesCommand(validIndex, validNote);
+
+        // Assert that the expected error is thrown
+        assertThrows(CommandException.class, AddNotesCommand.DUPLICATE_MESSAGE_CONSTRAINTS, () ->
+                addNotesCommand.execute(modelMock));
+    }
+
+    @Test
+    public void equals() {
+        Note noteAlice = new Note("Alice's note");
+        Note noteBob = new Note("Bob's note");
+        Index indexFirstPerson = Index.fromZeroBased(0);
+        Index indexSecondPerson = Index.fromZeroBased(1);
```

Comment:
Same thing maybe use this?
```suggestion
        Index indexFirstPerson = INDEX_FIRST_PERSON.fromZeroBased();
        Index indexSecondPerson = INDEX_SECOND_PERSON.fromZeroBased();
```